client/views/admin/users/EditUser.js

```diff
@@ -29,6 +29,8 @@ const getInitialValue = (data) => ({
 function EditUser({ data, roles, ...props }) {
 	const t = useTranslation();
 
+	const { reloadTable, ...otherProps } = props;
```

Comment:
I believe it's safe to do this destructuring at the function parameter declaration, i.e.:

```
export function AddUser({ roles, reloadTable, ...props }) {
```

I don't think `props` requires `reloadTable` property anywhere else.